src/components/MapLegendControl/MapLegend.tsx

```diff
@@ -25,7 +24,7 @@ function parseBlocks(
       if (value && Array.isArray(value) && value.length > 0) {
         const [name, ...args] = value;
 
-        switch (name as ExpressionName) {
+        switch (name as string) {
```

Comment:
There is an `Expression` type with a `match` operator. But I couldn't figure out how to use it the way we did originally. Given the this is just for TypeScript, I didn't think it was a blocker.

https://maplibre.org/maplibre-style-spec/expressions/